src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -2,6 +2,43 @@
 
 Create a new user in the Amazon Cognito UserPool by passing the new user's email address, password, and other attributes to `Auth.signUp`.
 
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+interface SignUpParams {
```

Comment:
@devnotfound, would you mind changing the name of the interface here to TSSignUpParams to preserve the name "SignUpParams" and ensure there's no duplicate declarations of it?